com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -315,6 +315,16 @@ private void Init(bool server)
             ConnectedClients.Clear();
             ConnectedClientsList.Clear();
 
+            // This is really only for editor unit tests that complete everything in a single frame
```

Comment:
Cool!  Couple of thoughts...

1. What if NetworkPrefabHandler is just created on demand on first get in a getter wrapper?  Then would that not cover both cases?
2. If not, strictly speaking should we not check if PrefabHandler is not null before instantiating it in the 'normal' place?

Reply:
I will refactor this a bit towards the instantiation pattern you described above.
Good thought!

Reply:
PrefabHandler now uses an "on demand" instantiation pattern:
_As long as there is a NetworkManager instance you should always be provided access to an instance of PrefabHandler._